NuGet.config

```diff
@@ -18,6 +18,9 @@
     <add key="dotnet5" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet5/nuget/v3/index.json" />
     <add key="dotnet5-transport" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet5-transport/nuget/v3/index.json" />
     <add key="darc-pub-dotnet-templating-247f60e9" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/darc-pub-dotnet-templating-247f60e9/nuget/v3/index.json" />
+    <!-- end for custom version of arcade -->
+    <!-- Developemt feed for Microsoft.Windows.SDK.NET.Ref -->
+    <add key="windows-sdk" value="C:\work\temp-cswinrt\feed" />
```

Comment:
Update that once the package is published to an non auth feed. Everything else is ready to review